src/test/java/com/wikia/webdriver/testcases/desktop/discussions/CreatingPostTests.java

```diff
@@ -94,21 +93,21 @@ public void userOnMobileCannotSavePostWithoutCategoryOrAnyContent() {
   }
 
   @Test(groups = MOBILE)
-  @Execute(asUser = User.USER, onWikia = MobileWikis.DISCUSSIONS_2)
+  @Execute(asUser = User.USER)
   @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
   public void postWithWeirdCharactersIsDisplayedOnMobilePostListPage() {
-    assertPostWithWeirdCharactersDisplayedOnPostsListPage(MobileWikis.DISCUSSIONS_2);
+    assertPostWithWeirdCharactersDisplayedOnPostsListPage("qadiscussions", "de");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "qadiscussions" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)